lib/ttcp

```diff
@@ -45,10 +48,16 @@ _TTCP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]:-${(%):-%N}}")"; cd ..; pwd)"
 # Dependent commands (non POSIX commands)
 _TTCP_DEPENDENCIES="${_TTCP_DEPENDENCIES:-yes openssl curl perl gzip}"
 
+# Algorithm for encoding data
+_TTCP_ENCRYPT_ALGORITHM="${_TTCP_ENCRYPT_ALGORITHM:-aes-256-cbc}"
+
 # Whare the last pasted content stored is.
 # It is re-used when you failed to get the remote content.
 TTCP_LASTPASTE_PATH_PREFIX="${TMPDIR}/lastPaste_"
+
 TTCP_ID_PREFIX="ttcopy"
+TTCP_SALT1="${TTCP_SALT1:-e7a082e7b396}"
```

Comment:
Adding short explanation like is kind 😎 :
`Salt used to encrypt the url of the file you uploaded. You can change it to any value  (more secure).`